src/js/components/stock-movement-wizard/StockMovementVerifyRequest.jsx

```diff
@@ -197,6 +197,9 @@ class StockMovementVerifyRequest extends Component {
             },
           };
 
+          const { pathname } = this.props.history.location;
+          this.props.history.replace({ pathname, search: `direction=${resp?.origin?.id === this.props.currentLocation?.id ? 'OUTBOUND' : 'INBOUND'}` });
```

Comment:
I do not get why this needs to be done. I think `create request` is always inbound and `verify request` is always outbound or if there is a way to enter verify request as `inbound` then why not add it as `injectedDirectly`?